tests/integration-tests/tests/common/utils.py

```diff
@@ -94,6 +94,29 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         raise
 
 
+def retrieve_pcluster_ami_with_additional_filters(os, architecture, additional_filters):
```

Comment:
if this does the same as `retrieve_latest_ami` why not adding the `filters` parameter to it and set a default to None?

Reply:
Done